deployment/conf/.templates/deployment.cfg.templ

```diff
@@ -0,0 +1,20 @@
+[KBaseSearchEngine]
+kbase-endpoint = {{ default .Env.kbase_endpoint "" }}
+job-service-url = {{ default .Env.job_service_url "" }}
+workspace-url = {{ default .Env.workspace_url "" }}
+shock-url = {{ default .Env.shock_url "" }}
```

Comment:
The vast majority of the service urls aren't needed

Reply:
True, but if someone wants to set them, they can put it into an environment variable instead of trying to hack up the container image.